services/BookService.cs

```diff
@@ -17,27 +17,28 @@ public class BookService(IFileService fileService) : IBookService
 
     private string filePath = FilePathsUtility.filePath;
 
-    public void RemoveBook()
+    public void RemoveBook(Book book)
     {
         // TODO: Move to void RemoveBook(Book book)
-        
-        // int chosenBook = BookHelper.GetIndexOfBookToModify("remove");
-        // List<string> lines = _fileService.ReadLinesFromFile(filePath).ToList();
-        // lines.RemoveAt(chosenBook);
-        // FileUtility.WriteLinesToFile(lines, filePath);
-        // Console.WriteLine("Book removed.");
-        // DisplayBooks();
+        var chosenBook = GetIndexOfBookToModify("remove");
+        var lines = _fileService.ReadLinesFromFile(filePath).ToList();
+        lines.RemoveAt(chosenBook);
+        _fileService.WriteLinesToFile(lines, filePath);
+        // the below line should be in InteractionController
+        Console.WriteLine("Book removed.");
+        DisplayBooks();
```

Comment:
`RemoveBook` is not concerned with Displaying Books, should not call `DisplayBook`